pxr/imaging/plugin/hdRpr/python/generateRenderSettingFiles.py

```diff
@@ -502,6 +502,17 @@ def houdini_parm_name(name):
             }
         ]
     },
+    {
+        'name': 'GMON',
+        'settings': [
+            {
+                'name': 'core:useGmon',
+                'ui_name': 'Use GMON',
+                'help': 'Enable fireflies suppression by using adaptive median of mean estimator',
```

Comment:
Thanks for adding the explanation.
It should probably also mention this is a Hybrid Pro only feature.

Reply:
Updated